### PR TITLE
feat(helm): add pgadmin4 as dependency to the chart

### PR DESCRIPTION
**Pull request by pmoscode**
## Description

This PR adds the pgadmin Helm chart as dependency to the MIW Helm chart as part of the series of PRs to implement issue #147 

Replaces #188 

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by borisrizov-zf** [comment]
I think we can add the `.helmignore`, update the `chart linting` and `helm-docs` commands to fix the failing checks. That should still keep the PR below 1000SLOC

**Comment by pmoscode** [comment]
Here is the original pgadmin chart and my PR to request the feature, I use here: https://github.com/rowanruseler/helm-charts/pull/232

File: charts/managed-identity-wallet/.helmignore [hunk 1]
```diff
@@ -24,5 +24,5 @@
 tests/
 values-*.yaml
 README.md.gotmpl
+.helmdocsignore
```
Comment (borisrizov-zf) [inline charts/managed-identity-wallet/.helmignore:27]:
```suggestion
.helmdocsignore
charts/
```

Comment (pmoscode) [reply charts/managed-identity-wallet/.helmignore:27]: Done

**Review by borisrizov-zf — APPROVED** [review]

**Pull request merged**